src/plugins/intel_gpu/src/graph/impls/ocl/reorg_yolo.cpp

```diff
@@ -46,14 +46,25 @@ struct reorg_yolo_impl : typed_primitive_impl_ocl<reorg_yolo> {
 namespace detail {
 
 attach_reorg_yolo_impl::attach_reorg_yolo_impl() {
-    implementation_map<reorg_yolo>::add(impl_types::ocl, reorg_yolo_impl::create, {
-        std::make_tuple(data_types::f32, format::bfyx),
-        std::make_tuple(data_types::f16, format::bfyx),
-        std::make_tuple(data_types::f32, format::yxfb),
-        std::make_tuple(data_types::f16, format::yxfb),
-        std::make_tuple(data_types::f32, format::byxf),
-        std::make_tuple(data_types::f16, format::byxf),
-    });
+    const auto types = {data_types::f16, data_types::f32};
+    const auto formats = {
+            format::bfyx,
+            format::yxfb,
+            format::byxf,
+            format::b_fs_yx_fsv16,
+            format::b_fs_yx_fsv32,
+            format::bs_fs_yx_bsv16_fsv16,
+            format::bs_fs_yx_bsv32_fsv16,
+            format::bs_fs_yx_bsv32_fsv32,
+    };
+
+    std::set<std::tuple<data_types, format::type>> keys;
+    for (const auto& t : types) {
+        for (const auto& f : formats) {
+            keys.emplace(t, f);
```

Comment:
Please use another `add` overload which will do the code above internally.

Reply:
Done.